### PR TITLE
Fixes #25511 - Disable max tasks by default to prevent memory issues.

### PR DESCRIPTION
**Pull request by chris1984**
https://projects.theforeman.org/issues/25511

### CODE REVIEW

**Comment by chris1984** [comment]
[test prprocessor]

File: spec/classes/katello_pulp_spec.rb [hunk 1]
```diff
@@ -44,7 +44,6 @@ class { '::katello':
             .with_enable_docker(true)
             .with_enable_ostree(false)
             .with_num_workers(1)
-            .with_max_tasks_per_child(2)
```
Comment (ekohl) [inline spec/classes/katello_pulp_spec.rb]: You can still test this and verify the value is `nil` instead of `2`

**Comment by chris1984** [comment]
@ekohl updated

**Comment by evgeni** [comment]
I have no clue about the actual :sparkle: number 2

**Pull request merged**